api/events/schema.py

```diff
@@ -148,6 +149,7 @@ def check_event_in_db(instance, info, event_check, **kwargs):
         event_id=kwargs['event_id']).scalar()
     if event and event_check == 'cancelled':
         event.cancelled = True
+        event.auto_cancelled = True
```

Comment:
What is the difference between a canceled event by the user from the google calendar and that one done by the mobile app after 10 minutes into the meeting?

Reply:
Event cancellation by the mobile app uses the `cancelEvent` mutation. This mutation is used only by the mobile app for `auto_cancellation`. From my understanding of the  `sync_all_events` method in  `helpers/calendar/events.py`, Events that were canceled from the calendar are updated in the DB as `archived` if they were already in the DB. If they are not already in the DB, they are not synced into the db at all.

Reply:
Does this mean that an event cannot be canceled using the mobile app unless it is auto-cancellation

Reply:
@mnswaleh Yes. You can only end an already started event from the mobile app. If you want to cancel an event, you will cancel it using google calendar.